packages/nextjs/pages/example-zk/GenerateProof.tsx

```diff
@@ -62,47 +76,89 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
   };
   return (
     <>
-      <div className="flex flex-col justify-center items-center">
-        <p>Generate proof-of-required-birthyear here. Copy/paste from prev window.</p>
-        <input
-          type="number"
-          placeholder="birth-year"
-          value={form.birthYear}
-          onChange={e => setForm({ ...form, birthYear: e.target.value as unknown as number })}
-        />
-        <input
-          type="number"
-          placeholder="required-birth-year"
-          value={form.requiredBirthYear}
-          onChange={e => setForm({ ...form, requiredBirthYear: e.target.value as unknown as number })}
-        />
-        <input
-          type="text"
-          placeholder="proof-of-birth-year-signed-message"
-          value={form.proofOfBirthYearSignedMessage}
-          onChange={e => setForm({ ...form, proofOfBirthYearSignedMessage: e.target.value })}
-        />
-        <input
-          type="text"
-          placeholder="proof-of-birth-year-public-key"
-          value={form.proofOfBirthYearPublicKey}
-          onChange={e => setForm({ ...form, proofOfBirthYearPublicKey: e.target.value })}
-        />
-        <AddressInput
-          value={form.personEthereumAddress}
-          name="personEthereumAddress"
-          placeholder="Ethereum address of person in signed message"
-          onChange={(val: string) => setForm({ ...form, personEthereumAddress: val })}
-        />
-        <button className="border-2 border-black" onClick={handleSubmission}>
-          Generate proof-of-required-birthyear
-        </button>
-        {proof && (
-          <div className="h-200 w-96 bg-green-50">
-            <p>Proof-of-required-birthyear:</p>
-            <p className="break-words">{proof}</p>
+      <p className="py-6">
+        {" "}
```

Comment:
A little bit of  hack to have this empty line here. 
Eventually would be good to add the proper top padding style to the paragraph tag.

Reply:
OK, I'm not sure about if there is a better way. But they use it in the original repo as well.